### ISSUE TITLE
Design & implement guidance pages

### ISSUE THREAD

**Issue by em-herrick** [GSA/notifications-admin#1868]
**Prototype / implement content changes**
- [ ] Create new Guidance Page in code, implement
- [ ] Implement changes to Using Notify 
- [ ] Delete/Hide Features page from Nav, in preparation for moving this before the login
- [ ] User test new static content experience with at least 4 users

**Comment by em-herrick** [comment]
Content document of guidance here: https://docs.google.com/document/d/1JgH4kKyVfnwh76tBCXjSvOtFk-6xeGVrs141_uAQ210/edit#heading=h.fz517f9xe4cf

**Comment by em-herrick** [comment]
At sprint refresh, we talked about getting @alexjanousekGSA in the mix to help implement some of these pages. More to come after the content is 'locked' by end of this week.

**Comment by heyitsmebev** [comment]
Edits/Feedback related to this epic (Guidance Sandbox Feedback) https://github.com/GSA/notifications-admin/issues/2050

**Comment by dmvancura** [comment]
asset for the 'Establish trust' page in the 'Get the word out'section[TextingBuckslip_English-vFF.pdf](https://github.com/user-attachments/files/17602352/TextingBuckslip_English-vFF.pdf)

**Comment by heyitsmebev** [comment]
@dmvancura, yup, I uploaded the new approved pdf to` Get the word out` section

**Comment by dmvancura** [comment]
research plan is here https://docs.google.com/document/d/12Aa9f4Yul_97U9umz9XD4K6Ziih1WKjD-uicJUxf8A8/edit?usp=sharing
research questions are here https://docs.google.com/document/d/1oTd0nbFzu6dqDL19aUHmw9IOOCig3nJCCW7dw2PStFM/edit?usp=sharing

**Comment by ecayer** [comment]
This looks great! Closing.
